### PR TITLE
Issue103

### PR DESCRIPTION
**Pull request by steingod**
This PR addresses the discussion of #103 and implements a free text field. Changes are made to mmd.xsd and the documentation for use_constraint.

### CODE REVIEW

**Comment by ferrighi** [comment]
Looks good to me. I only have a suggestion to include annotation in the schema as done in the documentation:
<xs:element name="license_text" type="xs:string" maxOccurs="1" minOccurs="0">
     <xs:annotation>
          <xs:documentation>This element holds the a free-text non formal license. If used identifier and resource should be left empty.</xs:documentation>
      </xs:annotation>
</xs:element>

**Comment by steingod** [comment]
Thanks, I have updated the XSD as suggested.

**Review by ferrighi — APPROVED** [review]
An extra xs:documentation closing tag has been left behind, which should be removed.

File: xsd/mmd.xsd [hunk 1]
```diff
@@ -142,6 +142,10 @@
         <xs:sequence>
             <xs:element name="identifier" type="mmd:use_constraint_identifier_type" maxOccurs="1" minOccurs="0"></xs:element>
             <xs:element name="resource" type="mmd:use_constraint_resource_type" maxOccurs="1" minOccurs="0"></xs:element>
+            <xs:element name="license_text" type="xs:string" maxOccurs="1" minOccurs="0"></xs:element>
+            <xs:annotation
+                <xs:documentation></xs:documentation>This element holds the a free-text non formal license. If used identifier and resource should be left empty.</xs:documentation>
```
Comment (mortenwh) [inline xsd/mmd.xsd]:
```suggestion
                <xs:documentation>This element holds the a free-text non formal license. If used identifier and resource should be left empty.</xs:documentation>
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
Looks good - only one suggestion to change something that seems like a bug...

File: xsd/mmd.xsd (hunk 1 above)
Comment (mortenwh) [reply xsd/mmd.xsd]: I see it was the same comment as Lara had :P - and it's solved now :)

**Review by mortenwh — APPROVED** [review]

**Pull request merged**